amethyst_renderer/src/formats/texture.rs

```diff
@@ -499,6 +513,64 @@ impl SimpleFormat<Texture> for TextureFormat {
     }
 }
 
+mod serde_helper {
+    use super::SamplerInfo as SamplerInfoComplex;
+    use serde::{Serialize, Deserialize, Serializer, Deserializer};
+    use tex::{FilterMethod, WrapMode};
+
+    #[derive(Serialize, Deserialize)]
+    #[serde(untagged)]
+    enum SamplerInfo {
```

Comment:
It's not a very clean solution, but I didn't see another way to allow specifying the filter without making it very verbose.

Reply:
This *is* clever solution.

Reply:
Actually, the easiest and best solution would be adding defaults to gfx.